src/BoardState.cpp

```diff
@@ -196,7 +200,7 @@ bool BoardState::authorize(card::uid_t uid)
             // User must leave the card for 3s before it's recognized
             delay(3000);
```

Comment:
how about a non-blocking wait? That would allow for a smoother UI to be implemented later. For example if the user decides to remove the card after 1 second can have an immediate feedback of his action 
For example a progress bar during the three seconds, could give a good visual feedback